src/Umbraco.Web.UI.NetCore/Startup.cs

```diff
@@ -6,6 +6,7 @@
 using Microsoft.Extensions.Hosting;
 using Umbraco.Cms.Core.DependencyInjection;
 using Umbraco.Extensions;
+using Smidge;
```

Comment:
Why is this committed?

Reply:
The thought behind this is that if you want to add app.UseSmidge() and setup some bundles. You would discover that extension method if you are poking around with `app.` with IntelliSense.

So for me its about discoverability. Happy to remove it though

Reply:
IMO you should do that in your own extension methods anyway and not directly in startup.cs. I think we should remove it.

Reply:
```suggestion

```